### ISSUE TITLE
按照processing 处理数据后开始训练，准确率一直不变，损失也变化不大,processing处理后的数据可以给出？

### ISSUE THREAD

**Issue by cqray1990** [tianchiguaixia/layoutlmv3-chinese#10]
20%|███████████████████████████████████                                                                                                                                            | 4636/23180 [19:11<1:04:57,  4.76it/s]{'loss': 1.4933, 'grad_norm': 9.056968688964844, 'learning_rate': 3.200172562553926e-05, 'epoch': 2.0}
                                                                                                                                                                                                                           {'eval_loss': 1.487217903137207, 'eval_precision': 0.004745470232959448, 'eval_recall': 0.00011145561027012787, 'eval_f1': 0.00021779590543697782, 'eval_accuracy': 0.37932501123828627, 'eval_runtime': 166.4599, 'eval_samples_per_second': 13.925, 'eval_steps_per_second': 13.925, 'epoch': 2.0}
 20%|███████████████████████████████████                                                                                                                                            | 4636/23180 [21:58<1:04:57,  4.76it/s]
python3.11/site-packages/transformers/modeling_utils.py:1742: FutureWarning: The `device` argument is deprecated and will be removed in v5 of Transformers.                          
  warnings.warn(
 30%|█████████████████████████████████████████████████████                                                                                                                            | 6954/23180 [30:14<56:23,  4.80it/s]{'loss': 1.489, 'grad_norm': 6.2279953956604, 'learning_rate': 2.800172562553926e-05, 'epoch': 3.0}
                                                                                                                                                                                                                           {'eval_loss': 1.5235638618469238, 'eval_precision': 0.004745470232959448, 'eval_recall': 0.00011145561027012787, 'eval_f1': 0.00021779590543697782, 'eval_accuracy': 0.37932501123828627, 'eval_runtime': 165.6114, 'eval_samples_per_second': 13.997, 'eval_steps_per_second': 13.997, 'epoch': 3.0}
 30%|█████████████████████████████████████████████████████                                                                                                                            | 6954/23180 [32:59<56:23,  4.80it/s]
/site-packages/transformers/modeling_utils.py:1742: FutureWarning: The `device` argument is deprecated and will be removed in v5 of Transformers.                          
  warnings.warn(
 30%|█████████████████████████████████████████████████████▊                                                                                                                           | 7042/23180 [33:27<56:48,  4.73it/s]

**Comment by wwwjo-ui** [comment]
想请教下，我的processing.py运行后，map进度不会推进，这是什么问题？
